src/ex1/index.html

```diff
@@ -1,14 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Exercise 1</title>
+    <title> Doneday App</title>
     <link rel="stylesheet" href="style.css" />
 </head>
-<body>
 
-<h1>Todo list</h1>
+<body onload="addImg()">
+
+    
+    <div class="container" id="div">
+        <h1> Doneday App </h1>
+        <div class="container-addTodo">
+            <form><input required type="text" class="taskText" placeholder="Add Your Task">
```

Comment:
nice usage of the form tag! it handles submit on enter & input validation for you